src/models/Polygon.js

```diff
@@ -21,16 +32,56 @@ export default class Polygon extends Shape {
       ...super.getInitialState(),
       ...{
         type: 'polygon',
-        polygonSides: 4
+        polygonSides: 4,
+        roundCorners: false,
+        roundFraction: 0.5,
```

Comment:
Would you consider making this a slightly lower value? I realize it's subjective, but I like 0.25 as the default value.